policies/adding-new-roles.md

```diff
@@ -0,0 +1,23 @@
+This document describes the process for adding new named roles. This
+document is *not* about how people are added to the roles, which is covered under
+[adding-new-role-members.md](https://github.com/astropy/astropy-project/blob/main/policies/adding-new-role-members.md).
+
+The process is:
+
+1. A voting member or maintainer in a related role, after consulting with other members in
+   related roles, opens a PR to [astropy.github.com](https://github.com/astropy/astropy.github.com)
+   to add this new role. The PR must describe clearly why this role is essential to the Astropy
+   Project and has clear description of associated responsibilities. If this role has sub-roles,
+   those must be clarified as well in the same manner. This new role must be declared Unfilled
+   for now (separation of concerns). This new role cannot duplicate any existing roles.
```

Comment:
This raises  the question how new roles may _overlap_ with existing ones. We probably don't want to simply infringe on those roles' responsibilities, but there can also be legit cases to split the tasks of one roles to several new ones.

Perhaps add something like
> If the new role touches or modifies the responsibilities of existing roles, those roles, and how they are affected by the proposed new role, shall also be named.

May also open the question of a policy for _removing_ roles in general.

Reply:
As for removing roles, you mean the assignments to a role, or the role itself, or both? Please clarify at https://github.com/astropy/astropy-project/issues/462 . Thanks!